tests/cover/test_find.py

```diff
@@ -42,7 +42,7 @@ def test_can_find_nan():
     find(floats(), math.isnan)
 
 
-def test_can_find_nans():
+def test_can_find_NaNs():
```

Comment:
I'm happy to use `_nan` instead. It's mostly the `s` at the end that bothers me / my dictionary.